### DISCUSSION TITLE
Convert asset account into a liability

### DISCUSSION THREAD

**Discussion by matejdro** [firefly-iii/firefly-iii#4500] [Firefly III]
Until recently I was using asset account with negative balance for my loan. But now I've noticed that Firefly has liability feature for exactly that.

Is there an easy way to convert asset account into a liability? Even if it involves manual database editing.

**Comment by JC5** [comment]
Afraid not, because the transaction types will also be different. If you have a backup however, you can change the `account_type_id` in the `accounts` table. Use the info in table `account_types` as a guide.

Then, run `php artisan firefly-iii:upgrade-database` and `php artisan firefly-iii:correct-database`.

It's a gamble but it should work.

**Reply by matejdro** [reply]
I have the backup but I don't want to spend the hassle of restoring it. Thanks for your help anyway.
